src/QueryDescription/test.js

```diff
@@ -635,4 +635,25 @@ describe('queryWithoutDeleted', () => {
       ]),
     )
   })
+
```

Comment:
place this test in the appropriate `describe` group - say, near line :400

we could also use a quick test in :451 (string check)